libs/compare_test.go

```diff
@@ -0,0 +1,58 @@
+package libs
+
+import (
+	"bytes"
+	"testing"
+)
+
+type compareTest struct {
+	i1     interface{}
+	i2     interface{}
```

Comment:
Why `interface{}` if it's always `[]byte`?

Reply:
I thought to use this structure for tests of other potential functions from the compare library

Reply:
all for the sake of extensibility)

Reply:
But we haven't planned any yet :)

Also, other compare functions may have different signature, for example with checking `err` response or have more arguments than 2. I think it's too premature for this kind of generalization

Reply:
so, now for testing compile we need wintun.dll) I guess, we should remove tests?

Reply:
sry, I go to bed and understand my mistake) fixed now